test/client.js

```diff
@@ -0,0 +1,16 @@
+import CozyClient from 'cozy-client'
```

Comment:
maybe this file needs to be moved in cozy-client

Reply:
Yep. Though I think that in the tests from Cozy-client, we already have stuff like this where we mock much more. So I think we're going to do this in cozy-client (move code inside cozy-client and export it) and then we're going to be able to import it here. I hope I am understandable.